testdata/expected/program/rails-sample/ruby.unit.json

```diff
@@ -1 +1 @@
-{"Name":".","Type":"ruby","Repo":"github.com/sgtest/rails-sample","Globs":null,"Files":["app/controllers/application_controller.rb","app/controllers/posts_controller.rb","app/helpers/application_helper.rb","app/helpers/posts_helper.rb","app/models/post.rb","config/application.rb","config/boot.rb","config/environment.rb","config/environments/development.rb","config/environments/production.rb","config/environments/test.rb","config/initializers/backtrace_silencers.rb","config/initializers/filter_parameter_logging.rb","config/initializers/inflections.rb","config/initializers/mime_types.rb","config/initializers/secret_token.rb","config/initializers/session_store.rb","config/initializers/wrap_parameters.rb","config/routes.rb","db/migrate/20140219041504_create_posts.rb","db/seeds.rb","test/controllers/posts_controller_test.rb","test/helpers/posts_helper_test.rb","test/models/post_test.rb","test/test_helper.rb"],"Dir":".","Dependencies":[["coffee-rails","~\u003e 4.0.0"],["jbuilder","~\u003e 1.2"],["jquery-rails","\u003e= 0"],["rails","= 4.0.3"],["sass-rails","~\u003e 4.0.0"],["sdoc","\u003e= 0"],["sqlite3","\u003e= 0"],["turbolinks","\u003e= 0"],["uglifier","\u003e= 1.3.0"]],"Data":{"files":["app/controllers/application_controller.rb","app/controllers/posts_controller.rb","app/helpers/application_helper.rb","app/helpers/posts_helper.rb","app/models/post.rb","config/application.rb","config/boot.rb","config/environment.rb","config/environments/development.rb","config/environments/production.rb","config/environments/test.rb","config/initializers/backtrace_silencers.rb","config/initializers/filter_parameter_logging.rb","config/initializers/inflections.rb","config/initializers/mime_types.rb","config/initializers/secret_token.rb","config/initializers/session_store.rb","config/initializers/wrap_parameters.rb","config/routes.rb","db/migrate/20140219041504_create_posts.rb","db/seeds.rb","test/controllers/posts_controller_test.rb","test/helpers/posts_helper_test.rb","test/models/post_test.rb","test/test_helper.rb"],"name":"rubyscripts"},"Ops":{"depresolve":null,"graph":null}}
```

Comment:
github.com/alexsaveliev/rails-sample -> github.com/sgtest/rails-sample, now that the other PR on that repository has been merged.